usher_tree/Snakefile

```diff
@@ -14,6 +14,7 @@ rule clean:
               viridian.topology.trees \
               viridian.mutations.trees \
               viridian.dated_samples.trees \
+              viridian.tsdated_samples.trees \
```

Comment:
Can we change this filename to "dates_nodes" actually while we're here please? I think that's a more descriptive name (and what I had in mind when I called the first one ``dated_samples``)

Reply:
Sure, good call.